modulefiles/gsi_orion.lua

```diff
@@ -1,20 +1,22 @@
 help([[
 ]])
 
-prepend_path("MODULEPATH", "/apps/contrib/NCEP/libs/hpc-stack/modulefiles/stack")
+prepend_path("MODULEPATH", "/work/noaa/epic-ps/role-epic-ps/miniconda3/modulefiles")
+miniconda3_ver=os.getenv("miniconda3_ver") or "4.12.0"
+load(pathJoin("miniconda3", miniconda3_ver))
 
-local hpc_ver=os.getenv("hpc_ver") or "1.1.0"
-local hpc_intel_ver=os.getenv("hpc_intel_ver") or "2018.4"
-local hpc_impi_ver=os.getenv("hpc_impi_ver") or "2018.4"
+prepend_path("MODULEPATH", "/work/noaa/epic-ps/role-epic-ps/hpc-stack/libs/intel-2022.1.2/modulefiles/stack")
```

Comment:
Does this `prepend_path` point at the official installation all UFS and EMC apps should use?

Reply:
@natalie-perlin confirmed this is the correct HPC-stack installation.

Reply:
Resolve with caveat noted above that g-w and library team staff need to confirm that machine specific modulefiles in this PR use officially maintained and supported modules.